python/tubeup/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           active_variants 1.1
+
+name                tubeup
+version             2023.8.19
+revision            0
+
+categories-append   multimedia
+license             GPL-3
+maintainers         nomaintainer
+
```

Comment:
looks like this port also doesn't install architecture-dependent files or compiles stuff. Please check, but if so it should also set "platforms" and "supported_archs".

Reply:
Basically if the code is compiled to native machine code, as opposed to ports that just install things like scripts and data files. You can check manually by running `file` on the files installed by a port, e.g.
```
% port -q contents py312-setuptools | xargs file | fgrep -i mach-o
% port -q contents zlib | xargs file | fgrep -i mach-o
/opt/local/lib/libz.1.3.1.dylib:                Mach-O 64-bit dynamically linked shared library x86_64
/opt/local/lib/libz.1.dylib:                    Mach-O 64-bit dynamically linked shared library x86_64
/opt/local/lib/libz.dylib:                      Mach-O 64-bit dynamically linked shared library x86_64
```
MacPorts 2.9 has a new feature where `port test` will check this for you, even if your port doesn't define any tests of its own. This is where the warnings annotations from GitHub Actions in the Files Changed tab of the PR come from.

Reply:
You can install the port and check the log and/or the files that get installed. For example, it files get compiled it will typically result in a `.so` file, then it will be differences depending on the OS version/architecture. One way to check is to check the downloadable file on PyPI, it it has `none-any.whl` in the filename it is likely not installing files that depend on the achitecture/OS version and thus
```platforns {darwin any}
supported_archs noarch
``` 
should be set